src/components/datasets/DatasetActivity/DatasetManifests.vue

```diff
@@ -113,6 +139,10 @@ export default {
       'getOrgMembersById'
     ]),
 
+    showManifests: function() {
```

Comment:
in future, can we use an arrow function expression instead of the function keyword? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Just a slightly more modern JS paradigm. 


```
showManifests: () => {}